src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import org.slf4j.LoggerFactory;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class ContentsUserMenu {
+
+    public static final String ERROR_INCORRECT_MENU_ITEM_SELECTED = "ERROR: Incorrect menu item selected \n";
+    public static final String ERROR_INVALID_NAME_FORMAT = "ERROR: Invalid name format!\n";
+    private static org.slf4j.Logger log = LoggerFactory.getLogger(ContentsUserMenu.class);
+
+
+    private HotelController hotelController;
+    private ReservationController reservationController;
+    private UserController userController;
+    private VisualUserMenu visualUserMenu;
+
+    public ContentsUserMenu(HotelController hotelController,
+                            ReservationController reservationController, UserController userController,
+                            VisualUserMenu visualUserMenu) {
+        this.hotelController = hotelController;
+        this.reservationController = reservationController;
+        this.userController = userController;
+        this.visualUserMenu = visualUserMenu;
+    }
+
+    public void mainMenu() {
+        VisualUserMenu.printListInConsole(ListMenu.getSiteHeader(), null);
+        VisualUserMenu.printListInConsole(ListMenu.getAuthorizationHeader(), ListMenu.getAuthorizationMenu());
+        Integer action;
+        do {
+            action = Integer.parseInt(visualUserMenu.getValidInputFromUser("Choose action: ", InputType.INTEGER));
+            switch (action) {
+                case (1):
+                    registrationUserMenu();
+                    break;
+                case (2):
+                    System.out.println("\tBye!");
+                    System.exit(0);
+                    break;
+                default:
+                    System.out.println(ERROR_INCORRECT_MENU_ITEM_SELECTED);
+                    break;
+            }
+        } while (!visualUserMenu.validateIntegerSize(ListMenu.getAuthorizationMenu().size(), action));
+    }
+
+    private void registrationUserMenu() {
+        String firstName = "";
+        String lastName = "";
+        boolean repeat = true;
+        do {
+            firstName = visualUserMenu.getValidInputFromUser("Enter your first name: ", InputType.STRING);
+            if (firstName.contains(" ") || firstName.isEmpty()) {
+                System.out.println(ERROR_INVALID_NAME_FORMAT);
+                continue;
+            }
+            lastName = visualUserMenu.getValidInputFromUser("Enter your last name: ", InputType.STRING);
+            if (lastName.contains(" ") || lastName.isEmpty()) {
+                System.out.println(ERROR_INVALID_NAME_FORMAT);
+                continue;
+            }
+            repeat = false;
+        } while (repeat);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)